### ISSUE TITLE
GCWeb top menu - Initial adjustment - Conformity with WCAG 2.1/2.2 Level AA

### ISSUE THREAD

**Issue by duboisp** [wet-boew/wet-boew#9445]
### SC 1.4.13 Content on hover or focus
* Evaluate and make the required adjustment.
  * There might be an issue when the focus are inside the menu items

### SC 2.4.11: Focus Appearance (Minimum)
* Evaluation and make the required adjustment

### SC 1.4.11: Non text content
* Fail F78
  * Reassess and make adjustment to avoid this failure
